guides/assets/override-solidus-assets.md

```diff
@@ -24,7 +24,8 @@ might not be effective for the next.
 
 If you want to tweak a CSS definition or change the way a particular JavaScript
 methods works, you can override it by redefining the definition in a file in
-your application's `app/assets` tree.
+your application's `app/assets` tree, although this isn't guaranteed to work in
+future versions of Solidus.
```

Comment:
As far as I read this paragraph it handles overriding a single CSS rule or JS function in a separate file, not overwriting the defining file itself. This is handled below. And overwriting things like this is always supported, no? Or am I misreading something?

Reply:
You're right I was confused about what this was describing. We should probably change this section (in a future PR) because "redefining" isn't an accurate description of what happens when adding additional CSS (and is likely misleading for JS too).

Reply:
:-1: My bad.